kibble/cli/commands/config_command.py

```diff
@@ -0,0 +1,31 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+__all__ = ["config_group"]
+
+import click
+
+
+@click.group(name="config")
+def config_group():
+    """Access configuration"""
+
+
+@config_group.command()
+def show():
+    """Shows configuration"""
+    click.echo("To be implemented!")
```

Comment:
Those are here just to show what we may want to do and how it can be structure. I can remove this but I think this may help newcomers to understand how the project is structured